leapmotion/README.md

```diff
@@ -0,0 +1,29 @@
+JavaCPP Presets for leapmotion
+===============================
+
+[![Gitter](https://badges.gitter.im/bytedeco/javacpp.svg)](https://gitter.im/bytedeco/javacpp) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/leapmotion/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/leapmotion) [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/https/oss.sonatype.org/org.bytedeco/leapmotion.svg)](http://bytedeco.org/builds/)  
+<sup>Build status for all platforms:</sup> [![leapmotion](https://github.com/bytedeco/javacpp-presets/workflows/leapmotion/badge.svg)](https://github.com/bytedeco/javacpp-presets/actions?query=workflow%3Aleapmotion)  <sup>Commercial support:</sup> [![xscode](https://img.shields.io/badge/Available%20on-xs%3Acode-blue?style=?style=plastic&logo=appveyor&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAMAAACdt4HsAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////////VXz1bAAAAAJ0Uk5T/wDltzBKAAAAlUlEQVR42uzXSwqAMAwE0Mn9L+3Ggtgkk35QwcnSJo9S+yGwM9DCooCbgn4YrJ4CIPUcQF7/XSBbx2TEz4sAZ2q1RAECBAiYBlCtvwN+KiYAlG7UDGj59MViT9hOwEqAhYCtAsUZvL6I6W8c2wcbd+LIWSCHSTeSAAECngN4xxIDSK9f4B9t377Wd7H5Nt7/Xz8eAgwAvesLRjYYPuUAAAAASUVORK5CYII=)](https://xscode.com/bytedeco/javacpp-presets)
```

Comment:
We should also add the workflow status badge to the main README.md as well as listing leapmotion as one of the available presets.